src/map/duel.hpp

```diff
@@ -12,18 +12,23 @@ struct duel {
 	int max_players_limit;
 };
 
-#define MAX_DUEL 1024 //max number of duels on server
-extern struct duel duel_list[MAX_DUEL]; //list of current duel
-extern int duel_count; //current number of duel on server
+//#define MAX_DUEL 1024 //max number of duels on server
+//extern struct duel duel_list[MAX_DUEL]; //list of current duel
+//extern int duel_count; //current number of duel on server
+
+duel& duel_GetDuelId( size_t did );
+bool duel_exist( size_t did );
+size_t duel_Getcount();
```

Comment:
duel_playercount() is not it better? or duel_count()?

Reply:
Duel_count. Sorry if misleading but this show the num of duel active not the number of player in duel

Reply:
then: duel_totalduels

or duel_countfights
or duel_countduels
or duel_countallduels
or duel_totalactive
or duel_countactive
or duel_countactiveduels
or duel_totalcount
or duel_counttotal
or duel_totalactiveduels

xD just suggestions 📟

Reply:
duel_counttotal() and maybe I'll add duel_countactive() for those that are being accepted.
(the 1st one just return when someone has created one but no fight is going on yet)